package.json

```diff
@@ -33,7 +33,8 @@
     "url": "https://github.com/SU-SWS/decanter/issues"
   },
   "dependencies": {
-    "bourbon": "^5.1.0"
+    "bourbon": "^5.1.0",
+    "custom-event-polyfill": "^1.0.7"
```

Comment:
Used to add data to the custom events.
https://www.npmjs.com/package/custom-event-polyfill